roles/https_haproxy/templates/ocsp.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Create OSCP Runner
+
+[Service]
+User=root
+Group=root
+Type=oneshot
+{% if enable_https and use_fixed_cert %}
+WorkingDirectory={{ cert_dir }}
+{% elif enable_https %}
+WorkingDirectory={{ letsencrypt.cert_dir }}
```

Comment:
This won't work for Letsencrypt certs, as those are not present in `{{ letsencrypt.cert_dir }}` but rather in `{{ letsencrypt.cert_dir }}/{{ hostname }}/`.
So either you need to adjust the `oscp.service` to iterate over all hosts (achievable with multiple Exec lines, I guess), or adjust the script to recurse the directory structure.